data/api/v2/full_spec.yaml

```
@@ -39018,6 +39024,15 @@ paths:
         schema:
           format: date-time
           type: string
+      - description: 'Boolean to specify whether to include accounts connected to
```

Does the reword make sense or am I missing the context?
'Boolean to include partner accounts connected to the current account in the Datadog partner network program. Defaults to `false`. '

```suggestion
      - description: 'Boolean to include partner accounts connected to
```